src/main/java/org/opentripplanner/graph_builder/module/OsmBoardingLocationsModule.java

```diff
@@ -125,6 +125,15 @@ private boolean connectVertexToStop(
         }
         new BoardingLocationToStopLink(ts, osmVertex);
         new BoardingLocationToStopLink(osmVertex, ts);
+        LOG.debug(
```

Comment:
```suggestion
        LOG.debug(
            "Connected {} ({}) to {} at {}",
            ts,
            stopCode,
            osmVertex.getLabel(),
            osmVertex.getCoordinate()
        );
```

SLF4J has a built in string interpolation feature.

Reply:
Oh, much better that way!